translations/monero-core_pl.ts

```diff
@@ -1515,68 +1522,68 @@ Uaktualnij go lub podłącz się do innego procesu</translation>
     <message>
         <location filename="../pages/Transfer.qml" line="69"/>
         <source>OpenAlias error</source>
-        <translation type="unfinished"></translation>
+        <translation>Błąd OpenAliasa</translation>
```

Comment:
Yes, but in Polish language you inflect nouns even if they are proper nouns. And that's whether people agree with it or not (some people think their names don't inflect which is a mistake). And it's the same with google. You say that you google something and we say "wygoogluję coś", "weź wygoogluj mi to", "muszę to wygooglować" or "googlnij mi to słowo".
To sum up, changing "OpenAliasa" to "OpenAlias" would be incorrect but if you prefer that, then let me know and I don't have any problems with that. I just wanted for you to know that I was sure when translating this phrase.

Reply:
I marked as optional since I don't speak Polish.  Feel free to leave it if you think _OpenAliasa_ is more clear for Polish speakers.  Seems Google will auto-correct to "OpenAlias" anyway  :-)